index.js

```diff
@@ -94,6 +96,51 @@ class PagedataRenderer {
       return allDone(null, reduction);
     });
   }
+
+  renderAndSave(projectSlug, templatePath, outputPath, allDone) {
+    const render = this.render.bind(this);
+    const pagedata = this.pagedata;
+    async.autoInject({
```

Comment:
since you are doing an eachSeries inside each function, I think it makes more sense to do getPages outside autoInject and then do an each with an autoInject inside.  What about using autoMap?  I wonder if we should remove the `reduce` part of autoMap - it's a little confusing.  So you just pass in function to get array, autoInject logic and then callback.  Thoughts?